.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Prepare release
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: new release version
+        required: true
+        default: (for example, 0.10.5)
+
+jobs:
+  release:
+    name: Prepare release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2.3.4
+
+      - name: Check input
+        run: |
+          if [[ ! "${{ github.event.inputs.release_version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo 'version "${{ github.event.inputs.release_version }}" not in ###.###.### format'
+            exit 1
+          fi
+          if [[ $( git ls-remote --tags origin refs/tags/v${{ github.event.inputs.release_version }} ) ]]; then
+            echo 'Tag "v${{ github.event.inputs.release_version }}" already exists. Double-check the version to release!'
+            exit 1
+          fi
+
+      - name: Create tag for release and prepare branch for next development version
+        run: |
+          git checkout -b release-v${{ github.event.inputs.release_version }}
+          git config user.email ${{ github.actor }}@users.noreply.github.com
+          git config user.name ${{ github.actor }}
+
+          ./mvnw --batch-mode versions:set versions:commit -DnewVersion=${{ github.event.inputs.release_version }}
+          git commit -am "preparing release ${{ github.event.inputs.release_version }}"
+          git tag v${{ github.event.inputs.release_version }}
+
+          ./mvnw --batch-mode org.apache.maven.plugins:maven-release-plugin:2.5.3:update-versions
+          git commit -am "next development version"
+
+          git push origin v${{ github.event.inputs.release_version }}
+          git push origin release-v${{ github.event.inputs.release_version }}
+
+      - name: Create pull request
+        uses: repo-sync/pull-request@v2.6
+        id: create-pr
+        with:
+          # Use a personal token to file a PR as a non-bot author to trigger other workflows (e.g., unit tests):
+          # https://docs.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token;
+          github_token: ${{ secrets.GA_RELEASE_PR_PERSONAL_TOKEN }}
```

Comment:
Would you add source code comment on how this is set?

Reply:
The link in the comment above explains it with details.

Reply:
I didn't find GA_RELEASE_PR_PERSONAL_TOKEN in https://docs.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token; Where did I miss?

Reply:
The doc has the link to "[Creating and storing encrypted secrets](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)", where they explain how one can create and store a secret, and access it in GitHub Actions. `GA_RELEASE_PR_PERSONAL_TOKEN` is just a variable name; you can name it however you want.

Reply:
Thank you for explanation. Then a source code comment "Save your access token as GA_RELEASE_PR_PERSONAL_TOKEN following http://..." is I was looking for.